solace/pom.xml

```diff
@@ -0,0 +1,224 @@
+             <tags>
+               <tag>
+                 <name>customTag1</name>
+                 <placement>a</placement>
+                 <head>Custom Tag One:</head>
+               </tag>
+               <tag>
+                 <name>customTag2</name>
+                 <placement>a</placement>
+                 <head>Custom Tag two:</head>
+               </tag>
+               <tag>
+                 <name>customTag3</name>
+                 <placement>a</placement>
+                 <head>Custom Tag three:</head>
+               </tag>
+             </tags>
+           </configuration>
+         </execution>
+       </executions>
+     </plugin>
+     <!-- Transform xml javadoc to stripped down version containing only class/interface comments and tags-->
+     <plugin>
+       <groupId>org.codehaus.mojo</groupId>
+       <artifactId>xml-maven-plugin</artifactId>
+       <version>1.0</version>
+       <executions>
+         <execution>
+           <id>transform-xmljavadoc</id>
+           <phase>generate-resources</phase>
+           <goals>
+             <goal>transform</goal>
+           </goals>
+         </execution>
+       </executions>
+       <configuration>
+         <transformationSets>
+           <transformationSet>
+             <dir>${project.build.directory}/generated-resources/xml-javadoc</dir>
+             <includes>
+               <include>${project.artifactId}-${project.version}-javadoc.xml</include>
+             </includes>
+             <stylesheet>XmlJavadocCommentsExtractor.xsl</stylesheet>
+             <outputDir>${project.build.directory}/generated-resources/xml-javadoc</outputDir>
+           </transformationSet>
+         </transformationSets>
+       </configuration>
+     </plugin>
+     <!-- copy xml javadoc to class jar -->
+     <plugin>
+       <artifactId>maven-resources-plugin</artifactId>
+       <version>2.6</version>
+       <executions>
+         <execution>
+           <id>copy-resources</id>
+           <phase>process-resources</phase>
+           <goals>
+             <goal>copy-resources</goal>
+           </goals>
+           <configuration>
+             <outputDirectory>${basedir}/target/classes</outputDirectory>
+             <resources>
+               <resource>
+                 <directory>${project.build.directory}/generated-resources/xml-javadoc</directory>
+                 <includes>
+                   <include>${project.artifactId}-${project.version}-javadoc.xml</include>
+                 </includes>
+                 <filtering>true</filtering>
+               </resource>
+             </resources>
+           </configuration>
+         </execution>
+       </executions>
+     </plugin>
+     <plugin>
+       <artifactId>maven-surefire-plugin</artifactId>
+       <configuration>
+         <argLine>-Xmx2048m</argLine>
+       </configuration>
+     </plugin>
+    </plugins>
+  </build>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.solacesystems</groupId>
```

Comment:
Please provide the references.

One potential issue is that users that are not aware of licensing restrictions unknowingly create restrictions for their own product, as it would be the case with xGPL.